google/pubsub_v1/services/publisher/client.py

```diff
@@ -713,6 +713,8 @@ def __init__(
             )
             # initialize with the provided callable or the passed in class
 
+            # TODO(https://github.com/googleapis/python-pubsub/issues/1349): Move the emulator
+            # code below to test files.
```

Comment:
I believe this file is autogenerated. We can modify the file by changing the code below

https://github.com/googleapis/python-pubsub/blob/8adb133a8caa0744980ffc393331fe4fdda2e3b4/owlbot.py#L102-L116